docs/docs/guides/sign-in-with-github-google-facebook-linkedin.mdx

```diff
@@ -98,8 +98,58 @@ selfservice:
 Next, open the login endpoint of the SecureApp and you should see the GitHub
 Login option!
 
-## Google, LinkedIn, Facebook
+## Google
+
+Go to [Google's developer docs here](https://developers.google.com/identity/sign-in/web/sign-in) and configure a project:
+
+- Select "Web server" as a caller app.
+- Set "Authorized redirect URIs" to `http://127.0.0.1:4455/.ory/kratos/public/self-service/browser/flows/strategies/oidc/callback/google`.
+- Make sure to copy over the client id and secret to use in your configurations file.
+
+Your Jsonnet code snippet for the provider that only uses email address would look like this (you can place it in the same directory as your github one):
+
+```json title="contrib/quickstart/kratos/email-password/oidc.github.jsonnet"
+local claims = std.extVar('claims');
+
+{
+  identity: {
+    traits: {
+      email: claims.email
```

Comment:
It probably makes sense to copy this section from Github: https://github.com/ory/kratos/pull/424/files#diff-5beaed0e278de0cfec10f48986724b53R63-R76

Reply:
@aeneasr do you the comment about verified email address? Would be relevant for google as well?

Reply:
Yes it is also relevant for Google!